.github/workflows/build-test.yml

```diff
@@ -20,12 +20,22 @@ jobs:
     runs-on:                    ${{ matrix.platform }}
     steps:
       - name:                   Checkout sources
-        uses:                   actions/checkout@v2
+        uses:                   actions/checkout@master
         with:
-          fetch-depth:          50
-      - name:                   Checkout submodules
-        shell:                  bash
-        run:                    git submodule update --init --recursive
+          submodules:           true
+      # https://github.com/actions/cache/issues/133 
+      - name:                   Fixup the owner of ~/.cargo/
```

Comment:
interesting, I don't find information on github actions about specifing different PID for running the actions. How is it that scripts create files with different PID?

Reply:
`~/.cargo/` is owned by root: https://github.com/bjorn3/rustc_codegen_cranelift/runs/508021777?check_suite_focus=true#step:3:4

Reply:
the problem is likely how actions set up the rust environment
https://github.com/actions/cache/issues/133#issuecomment-569388746
https://github.com/actions/virtual-environments/blob/master/images/win/scripts/Installers/Install-Rust.ps1

Reply:
Filed https://github.com/actions/virtual-environments/issues/572